src/SubGraphFunctions.jl

```diff
@@ -87,6 +87,45 @@ function buildSubgraphFromLabels(dfg::G,
 
   return cliqSubFg
 end
+
+#TODO solvable
+function buildSubgraphFromLabels!(dfg::AbstractDFG,
```

Comment:
PR is more about this function. Note its implemented and used in parametric

Reply:
also, does this function replace existing functionality already in DFG?  buildSubGraph is a DFG thing, no?  There used to be a `buildSubGraph` in IIF, and then we moved it to DFG -- this looks like you built a new one putting it back in IIF.

@GearsAD, do you have any feelings about this in IIF.  My feeling is subGraph build functions are set-theory operations that belong in DFG?

Reply:
for example these functions in DFG?
- https://github.com/JuliaRobotics/DistributedFactorGraphs.jl/blob/258cc40332910e97dd0ca932170d28b85e7aecbf/src/GraphsDFG/services/GraphsDFG.jl#L337
- https://github.com/JuliaRobotics/DistributedFactorGraphs.jl/blob/258cc40332910e97dd0ca932170d28b85e7aecbf/src/LightDFG/services/LightDFG.jl#L394

Reply:
my suggestion is to move buildSubGraphsFromLabels up to DFG